modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -182,4 +182,6 @@ AssetHelper assetHelper = (AssetHelper)request.getAttribute(AssetWebKeys.ASSET_H
 Format dateFormatDate = FastDateFormatFactoryUtil.getDate(locale, timeZone);
 %>
 
-<%@ include file="/init-ext.jsp" %>
\ No newline at end of file
+<%@ include file="/init-ext.jsp" %>
+
+<liferay-frontend:stylesheet bundle="com.liferay.frontend.theme.classic.style.guide.sample.web" css="css/edu.css" />
```

Comment:
We need to import the new `file.css` (created in another module) into our module
- use https://github.com/liferay/liferay-portal/blob/master/modules/modules.properties to find the right `bundle` name
- pay attention to the folder inside the module (in my case it's called `/css/`)